src/methods/send.ts

```diff
@@ -93,4 +88,52 @@ const send = async (
   }
 };
 
+const setFocusObject = (action: BrowserSetFocusAction, genomeBrowser: GenomeBrowserType) => {
+  const { focusType } = action.payload;
+
+  if (focusType === 'gene') {
+    setFocusGene(action, genomeBrowser);
+  } else if (focusType === 'location') {
+    setFocusLocation(action, genomeBrowser);
+  }
+};
+
+const setFocusGene = (action: BrowserSetFocusAction, genomeBrowser: GenomeBrowserType) => {
+  const { genomeId, focusId, bringIntoView } = action.payload;
+
+  if (bringIntoView) {
+    genomeBrowser.jump(`focus:${genomeId}:${focusId}`);
+    genomeBrowser.wait();
+  }
+
+  genomeBrowser.switch(['track', 'focus'], true);
+  genomeBrowser.switch(['track', 'focus', 'label'], true);
+  genomeBrowser.switch(['track', 'focus', 'item', 'gene'], {
+    'genome_id': genomeId,
+    'item_id': focusId
+  });
+};
+
+const setFocusLocation = (action: BrowserSetFocusAction, genomeBrowser: GenomeBrowserType) => {
+  // NOTE: This is a temporary function, until the genome browser is released with proper support of genome locations
+  const { genomeId, focusId, bringIntoView } = action.payload;
+  const locationRegex = /(.+):(\d+)-(\d+)/;
+  const [, regionName, start, end] = locationRegex.exec(focusId) ?? [];
+  if (!regionName && start && end) {
```

Comment:
What if `start` and/or `end` is not there?

Reply:
The code will return on the next line without doing anything. Why?

Reply:
Oh, that's not what my conditional was saying; I see it now. Updated.